src/engine/render.tsx

```diff
@@ -1,12 +1,33 @@
-import type { IsomerPageSchema } from "~/engine"
-import { renderLayout as renderNextLayout } from "~/templates/next"
+import type { IsomerMetaHeadSchema, IsomerPageSchema } from "~/engine"
+import {
+  renderLayout as renderNextLayout,
+  renderComponent as renderNextComponent,
+} from "~/templates/next"
 
 export const RenderEngine = (props: IsomerPageSchema) => {
-  if (props.site.theme === "next") {
+  if (props.site.theme === "isomer-next") {
     return renderNextLayout(props)
   }
 
   return null
 }
 
+export const RenderMetaHead = (props: IsomerMetaHeadSchema) => {
```

Comment:
actually I'm getting more confused LOL, don't quite understand why we need this `RenderMetaHead` method

To me it seems more intuitive to pass in the `HeadComponent` to `RenderEngine` (it already takes in the other props needed anyway) and then pass those to the `Skeleton` layout which can then use `HeadComponent` to inject whatever meta tags are required for that page. Maybe I'm tripping, lol pls correct me


![Screenshot 2024-03-23 at 1 13 45 AM](https://github.com/isomerpages/isomer-components/assets/77665182/ea82dae0-aede-4a96-859f-169249fb29f4)

Reply:
Yeah hmm I suppose that is possible, the only reason why I decided to separate it out is because this would be the only instance that uses the `<Head>` component, so I kind of wanted to avoid bringing it through the entire RenderEngine process.

I got no strong opinions on this one, am ok either way.

Reply:
Clarified offline, agree with the approach of having only a single entry point. Updated in 3f670f8d2401008298225e9d032257d6eebf8c46.